src/Symfony/Bridge/PhpUnit/Legacy/TestRunnerForV6.php

```diff
@@ -24,7 +24,7 @@ class TestRunnerForV6 extends BaseRunner
     /**
      * {@inheritdoc}
      */
-    protected function handleConfiguration(array &$arguments): void
+    protected function handleConfiguration(array &$arguments)
```

Comment:
The existing runner looses its void return type, but keeps the code changes (doesn't return anything, extends from `PHPUnit\TextUI\TestRunner`)